packages/admin-panel-server/src/viz-builder/dashboardVisualisation/combineVisualisation.ts

```diff
@@ -4,15 +4,12 @@
  */
 
 import { DashboardItem, LegacyReport, Report, DashboardVizResource, DashboardViz } from '../types';
+import { extractFetch } from './helpers';
 
 const getData = (report: Report) => {
   const { config } = report;
-  const {
-    fetch: { dataElements, dataGroups, aggregations },
-    transform,
-  } = config;
-
-  return { dataElements, dataGroups, aggregations, transform };
+  const { fetch, transform } = config;
+  return { ...extractFetch(fetch), transform };
```

Comment:
Imported/exported viz config looks like the following:

```js
{
  // ...
  "data": {
    "aggregations": [],
    "dataGroups": ["SCRF"],
    "startDate": { "unit": "day", "offset": -6 },
    "transform": [],
  }
}
```

I feel it would be preferrble to
* group fetch params and separate them from `transform`
* instead of `data`, name this field `config` or `reportConfig`

In general, with the exception of `name` (which is pulled out of `dashboard_item.config`) I feel it's fine for the report config fields to retain their DB structure.

However, I felt that changing the above was out of scope here (if we agree that this would be an improvement)